gspread-project/scripts/tests/test_first.py

```diff
@@ -0,0 +1,69 @@
+import os
+from push_data import *
+import pandas as pd
+import gspread
+from dotenv import load_dotenv
+import json
+from unittest.mock import MagicMock
+
+def test_service_file_path():
+    """
+    This test the service file path declared in .env if it contains neccessary keys
+    """
+    load_dotenv()
+    path = os.getenv("service_file_path")
+    with open(path) as jsonFile:
+        data = json.load(jsonFile)
+        keys = data.keys()
+
+        assert 'private_key_id' in keys
+        assert 'project_id' in keys
+        assert 'client_id' in keys
+        assert 'token_uri' in keys
+        assert 'auth_uri' in keys
+
+def test_create_spreadsheet():
+    """
+    tests the create spreadsheet function
+    """
+
+    load_dotenv()
+    service = os.getenv("service_file_path")
+    title = os.getenv("title")
+    gc = gspread.service_account(service)
```

Comment:
Don't do this. When writing tests, a good practice is to avoid anything that will call APIs or connect to an external network outside your environment. So the way to go is to use mocks, I see you did that in some other places already. If you try to run a test that connects over an API you're not unit-testing your code, because you're now testing the code of the API inside your own code, the goal of this project is unit testing not integration testing